packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/ZoomEventLayer/ZoomEventLayer.js

```diff
@@ -34,33 +37,53 @@ function ZoomEventLayer ({
   panning = false,
   top = 0,
   width,
+  children,
   ...rest
 }) {
+  const root = useRef(null)
   const theme = useTheme()
   const focusColor = theme?.global.colors[theme.global.colors.focus]
+
+  const handleWheel = disabled ? DEFAULT_HANDLER : ({ event }) => onWheel(event)
+  useWheel(handleWheel, {
+    eventOptions: { passive: false },
+    target: root
+  })
+
+  function handlePointerFocus(event) {
+    console.log('handlePointerFocus')
+    if (disabled) return false
+    root.current?.focus()
```

Comment:
This will scroll back to the top of the page for transcription projects. 

```suggestion
    root.current?.focus({
      preventScroll: true
    })
```